appveyor-nightly.yml

```diff
@@ -45,6 +45,7 @@ before_build:
 after_build:
   - rake "create_setup[%APPVEYOR_BUILD_VERSION%, %CONFIGURATION%, C:/smartxls.zip, 2.6.2.9]"
   - 7z a setup.zip %APPVEYOR_BUILD_FOLDER%\setup\deploy\*.msi
+  - rake "create_portable_setup[%APPVEYOR_BUILD_VERSION%, %CONFIGURATION%, pksim-portable-setup.zip]"
```

Comment:
create_setup creates a msi.
create_portable creaes a zip
It's symmetrical in that case